cmd/cli/reconcile.go

```diff
@@ -113,10 +113,10 @@ func runReconciler(cmd *cobra.Command, args []string) {
 
 		blocks := []*types.Block{}
 		for i := curr; i <= to; i++ {
-			block, err := fetcher.BlockRetry(ctx, network, &types.PartialBlockIdentifier{
+			block, fetcherErr := fetcher.BlockRetry(ctx, network, &types.PartialBlockIdentifier{
 				Index: &i,
 			})
-			utils.ExitOnError(err)
+			utils.ExitOnError((*fetcherErr).Err)
```

Comment:
should this just take the fetcherErr itself? I assume it's nil if there's no error

Reply:
this is weird--so fetcherErr is now no longer of type `error` which means it can't take it directly, but then this being `nil` poses an issue. I modified it and extracted it out in case that error format changes further in the future.